### DISCUSSION TITLE
Suggestion: radically simplify the CLA

### DISCUSSION THREAD

**Discussion by DePasqualeOrg** [ory/kratos#2774] [Ideas]
To lower the barrier to contributing to Ory's repositories, I'd like to suggest radically simplifying the CLA that contributors are required to sign. Ideally it should be just a few sentences in simple language. In its current state, I think it's likely that most people will not read through the entire agreement, and will either decide not to contribute, or will blindly accept without a full understanding of the contents.

**Comment by vinckr** [comment]
Hey Anthony, 
happy to look into anything that lowers the barrier to contributions! 
Do you have an example for such a CLA maybe? I think the one we have right now ([source btw](https://gist.github.com/aeneasr/1c801494d360c9234def4952d4fb3da8)) is a fairly standard one. 
Then again i am not a lawyer, so not sure what is exactly required.
Essentially the CLA makes sure that we can make use of contributions under our [license](https://github.com/ory/kratos/blob/master/LICENSE) without fear of getting sued down the line.
